src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Kafka/KafkaConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Kafka
+{
+    internal static class KafkaConstants
+    {
+        internal const string IntegrationName = nameof(IntegrationIds.Kafka);
+        internal const string ConsumeOperationName = "kafka.consume";
+        internal const string ProduceOperationName = "kafka.produce";
+        internal const string TopicPartitionTypeName = "Confluent.Kafka.TopicPartition";
+        internal const string MessageTypeName = "Confluent.Kafka.Message`2[!0,!1]";
+        internal const string ConsumeResultTypeName = "Confluent.Kafka.ConsumeResult`2[!0,!1]";
+        internal const string ActionOfDeliveryReportTypeName = "System.Action`1[Confluent.Kafka.DeliveryReport`2[!0,!1]]";
+        internal const string TaskDeliveryReportTypeName = "System.Threading.Tasks.Task`1[Confluent.Kafka.DeliveryReport`2[!0,!1]]";
+        internal const string DeliveryReportOfNullTypeName = "Confluent.Kafka.DeliveryReport`2[Confluent.Kafka.Null,Confluent.Kafka.Null]";
```

Comment:
I didn't see this being used anywhere, can it be removed?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/119930f7ef7af6994f9d59c0debc1cea8d7847b6